consensus/core/src/context.rs

```diff
@@ -102,23 +102,43 @@ impl Context {
 /// `[Clock]` cloneable to ensure that a single instance is shared behind an
 /// `[Arc]` wherever is needed in order to make sure that consecutive calls to
 /// receive the system timestamp will remain monotonically increasing.
-pub(crate) struct Clock {
+pub struct Clock {
     initial_instant: Instant,
     initial_system_time: SystemTime,
+    // `clock_drift` should be used only for testing
```

Comment:
if clock drift is only used for testing, then use `#[cfg(test)]` for this field, as well as `#[cfg(not(test))]` for the standard `Default` implementation as well as `#[cfg(test)]` for the test `default` implementation